libecho/src/IReceiver.h

```diff
@@ -0,0 +1,5 @@
+class IReceiver {
+public:
+	virtual int receive(char *buffer, int size) = 0;
+private:
+};
```

Comment:
We may also need getters and setters for, for example, windowSize, it would be easier to declare them here than in every Receiver class.